templates/server/puppet.conf.erb

```diff
@@ -15,10 +15,7 @@
 <% end -%>
     strict_variables = <%= scope.lookupvar("puppet::server_strict_variables") %>
 
-<% if scope.lookupvar("puppet::server_directory_environments") -%>
-    environmentpath  = <%= scope.lookupvar("puppet::server_envs_dir") %>
-    basemodulepath   = <%= [scope.lookupvar("puppet::server_common_modules_path")].flatten.join(':') %>
-<% elsif scope.lookupvar("puppet::server_git_repo") ||
+<% if scope.lookupvar("puppet::server_git_repo") ||
```

Comment:
You got good point there. I've commit and squash all commits. Can you check again?
Travis CI is happy but we might add this check for the future.